packages/sn-filter-pane/src/components/FoldedListbox/DimensionIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import { Tooltip } from '@mui/material';
+import React from 'react';
+import { stardust } from '@nebula.js/stardust';
+import DrillDown from './drillDown';
+import Cyclic from './cyclic';
+
+interface Props {
+    isRtl: boolean;
+    translator?: stardust.Translator;
+    type: string;
+}
+
+const DimensionIcon = ({
+  type, translator, isRtl,
+}: Props) => {
+  const delay = 2000;
+  const style = { fontSize: '12px', padding: isRtl ? '0 0 0 6px' : '0 6px 0 0' };
+  let title;
+  let icon;
+  switch (type) {
+    case 'H':
+      icon = (<DrillDown style={style} />);
+      title = translator?.get('Listbox.DrillDown');
+      break;
+    case 'C':
+      icon = (<Cyclic style={style} />);
+      title = translator?.get('Listbox.Cyclic');
+      break;
+    default:
+      return undefined;
+  }
+  return (<Tooltip title={title} enterDelay={delay}>
```

Comment:
Should we return undefined if type is not 'H' or 'C'.

Reply:
Yes if type is not H or C it has no icon which we return undefined

Reply:
With the current code, if type is not H or C it will return
```
(<Tooltip title={title} enterDelay={delay}>
    <div>
   </div>
</Tooltip>)
```
Is that what we want?

Reply:
I mean instead of return 
```
(<Tooltip title={undefined} enterDelay={delay}>
    <div>
   </div>
</Tooltip>)
```
we just return undefined in case of H or C

Reply:
No we don't. It returns immediately after the `switch default` when it is not H or C which means no code after line 31 is executed in case it goes into the `default` case.

Reply:
@quanho What do you think? Is it ok :)?